lib/context.js

```diff
@@ -160,8 +160,7 @@ const proto = module.exports = {
   get cookies() {
     if (!this[COOKIES]) {
       this[COOKIES] = new Cookies(this.req, this.res, {
-        keys: this.app.keys,
-        secure: this.request.secure
```

Comment:
It's not a bug.

Reply:
Why detect secure by default? It is a problem in my project, even if it is easy to solve.

The document does not show `secure detect`, Other developers will encounter this problem.